presto-native-execution/src/test/java/com/facebook/presto/spark/AbstractTestNativeSimpleQueries.java

```diff
@@ -39,8 +39,8 @@
  * <p>
  * For test queries requiring shuffle, the disk-based local shuffle will be used.
  */
-public class TestPrestoSparkNativeExecution
-        extends AbstractTestPrestoSparkQueries
+public abstract class AbstractTestNativeSimpleQueries
```

Comment:
Do we need this? Ideally, we have a shared set of queries. We can move testJsonFileBasedFunction into a spark-specific test suite.

Reply:
My plan is before we cover a relatively large set of Prestissimo tests, we shall still keep these tests around for at least some coverage. Then we can remove it maybe after a couple of weeks.